### ISSUE TITLE
Efficient Learning of Control Policies for Robust Quadruped Bounding using Pretrained Neural Networks. (arXiv:2011.00446v2 [cs.RO] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#8383]
Efficient Learning of Control Policies for Robust Quadruped Bounding using Pretrained Neural Networks. (arXiv:2011.00446v2 [cs.RO] UPDATED)<br>
https://ift.tt/3jOWoOd<br>
<p>Bounding is one of the important gaits in quadrupedal locomotion for negotiating obstacles. However, due to a large number of robot and environmental constraints, conventional planning and control has limited ability to adapt bounding gaits on various terrains in real-time. We proposed an efficient approach to learn robust bounding gaits by first pretraining the deep neural network (DNN) using data from a robot that used conventional model-based controllers. Next, the pretrained DNN weights are optimized further via deep reinforcement learning (DRL). Also, we designed a reward function considering contact points to enforce the gait symmetry and periodicity, and used feature engineering to improve input features of the DRL model and the bounding performance. The DNN-based feedback controller was learned in simulation first and deployed directly on the real Jueying-Mini robot successfully, which was computationally more efficient and performed much better than the previous model-based control in terms of robustness and stability in both indoor and outdoor experiments.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
